pg_mapper_derive/src/lib.rs

```diff
@@ -97,11 +97,7 @@ fn impl_tokio_pg_mapper(
                 })
             }
 
-            fn from_rows(rows: Vec<tokio_postgres::row::Row>) -> ::std::result::Result<Vec<Self>, tokio_pg_mapper::Error> {
-                rows.iter().map(|row| Self::from_row(row).map_err(|e| e.into())).collect()
-            }
-
-            fn from_rows_ref(rows: Vec<&tokio_postgres::row::Row>) -> ::std::result::Result<Vec<Self>, tokio_pg_mapper::Error> {
+            fn from_rows(rows: Vec<&tokio_postgres::row::Row>) -> ::std::result::Result<Vec<Self>, tokio_pg_mapper::Error> {
```

Comment:
Shouldn't this be `Vec<Row>` ?  That way, the result of `query` will feed into `from_rows`:  https://docs.rs/tokio-postgres/0.5.4/tokio_postgres/struct.Client.html#method.query

Reply:
@godofdream ^